.changeset/blue-wombats-hope.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage/plugin-kubernetes-backend': minor
+'@backstage/plugin-kubernetes-common': minor
+---
+
+The Kubernetes Builder service now makes use of a Permission Evaluator field which was introduced to the KubernetesEnvironment interface and subsequently used throughout the builder.
```

Comment:
If i follow, this is technically a breaking change, since we're requiring more things as parameters, right? Both modules are on major version 0 so I agree with the minor. 

However, i think it would be nice to readers of the changelog to clearly state that they have to add the parameter.

Reply:
Yes you are correct. A permission Evaluator field would now be required as an additional parameter. Thanks for the feedback i can definitely make this change-log a bit clearer.

Reply:
@agentbellnorm Added a new line to the changeset, hope that its clear enough. If you were thinking something more specific let me know. Thanks